### PR TITLE
Make the build server Windows script parallel

### PR DESCRIPTION
**Pull request by borrrden**

### CODE REVIEW

File: jenkins/build_server_win.ps1 [hunk 1]
```diff
@@ -150,16 +154,21 @@ function Run-UnitTest() {
     Pop-Location
 }
 
-foreach ($arch in @("Win32", "Win64", "ARM")) {
+$ArtifactsDir = "$env:WORKSPACE\artifacts\$($ShaVersion.substring(0, 2))\$ShaVersion"
```
Comment (mhocouchbase) [inline jenkins/build_server_win.ps1:157]: does $ArtifactsDir need to be created first or Copy-Item will automatically create it?

**Review by mhocouchbase — COMMENTED** [review]
need to add "--parallel $Parallel" to cmake to build in Parallel.

**Comment by mhocouchbase** [comment]
The naming convention for debug zips in this script look more consistent than what was in the bat script.  Need to make sure the downstream consumer is aware of the name change.
%PRODUCT%-%VERSION%-%SHA_VERSION%-%OS%-win32-debug.zip
%PRODUCT%-%VERSION%-%SHA_VERSION%-%OS%-debug-arm.zip
%PRODUCT%-%VERSION%-%SHA_VERSION%-%OS%-win64-winstore-debug.zip

File: jenkins/build_server_win.ps1 (hunk 1 above)
Comment (borrrden) [reply jenkins/build_server_win.ps1:157]: It is created on the line below this one actually

**Comment by borrrden** [comment]
What is the `$Parallel` value?

**Comment by borrrden** [comment]
Noted on the names

**Review by mhocouchbase — APPROVED** [review]

**Pull request merged**